pipeline/builder.go

```diff
@@ -201,5 +201,23 @@ func getMiddlewares(config Config) []endpoint.Middleware {
 		e = append(e, stalemiddleware.New(c))
 		log.Warn().Msgf("[goduck][pipeline] Stale middleware is active. The system will be set to unhealthy if no message is received in %s.", config.StaleAfter.String())
 	}
+
+	if config.InputStream.DLQKafkaTopic != "" && config.Backoffmiddleware.MaxRetries == backoffmiddleware.MaxRetriesInfinite {
+		log.Warn().Msgf("[goduck][pipeline] DLQ is active but max retries is set as infinite. This may cause a infinite loop if the returned service error with wrigth severity was not SeverityInput type.")
```

Comment:
I think there is a typo at `wrigth`

Also `a infinite` should be `an infinite`

Suggestion: Configuring a DLQ with infinite retries should raise a configuration error. This should not be allowed because can mislead the developers in thinking that there is a DLQ for this worker when in fact the is not.

Reply:
Done